### PR TITLE
intersphinx xref physical types

### PR DESCRIPTION
**Pull request by nstarman**
Signed-off-by: Nathaniel Starkman (@nstarman) <nstarkman@protonmail.com>

### Describe your changes

Ahh. This solution will require astropy v4.3+ for the physical types. ~I'll get back to this later, when I update my PR to sphinx-astropy.~
~Done. Now need to get RTD to compile with astropy v4.3.   hmm.... any suggestions?~
~without building RTD in astropy v4.3+, the links are not live. https://gala-astro--223.org.readthedocs.build/en/223/~

Current state : Gala doesn't work with astropy dev #222

### Checklist

* [ ] Did you add tests?
* [ ] Did you add documentation for your changes?
* [ ] Did you reference any relevant issues?
* [ ] Did you add a changelog entry? (see `CHANGES.rst`)
* [ ] Are the CI tests passing?
* [ ] Is the milestone set?

Requires: #224

### CODE REVIEW

**Comment by nstarman** [comment]
Ahh. This solution will require astropy v4.3+ for the physical types. I'll get back to this later, when I update my PR to sphinx-astropy.

File: .rtd-environment.yml [hunk 1]
```diff
@@ -3,4 +3,5 @@ dependencies:
   - gsl
   - pip
   - pip:
-    - -e .[docs]
+    - astropy  @ git+https://github.com/astropy/astropy.git
```
Comment (nstarman) [inline .rtd-environment.yml:6]: need to figure out a better way of setting astropy >= 4.3

File: setup.cfg [hunk 2]
```diff
@@ -69,7 +69,7 @@ docs =
     sphinxcontrib-bibtex
     sphinx-astrorefs
     sphinx_automodapi
-    sphinx_astropy
+    sphinx_astropy   @ git+https://github.com/nstarman/sphinx-astropy.git@intersphinx-xref-physical-types
```
Comment (nstarman) [inline setup.cfg:72]: TODO: change when PR merged.

File: .rtd-environment.yml (hunk 1 above)
Comment (nstarman) [reply .rtd-environment.yml:6]: @adrn, any suggestions?

Comment (adrn) [reply .rtd-environment.yml:6]: Fine to pin this as is for now. I want to hold off on requiring >= 4.3 until gala v1.5 (next release is v1.4, hopefully within the next few weeks). By then, 4.3 should be out and we can change this to astropy>=4.3

**Comment by adrn** [comment]
Thanks for starting on this! I updated the conf.py in #243 and think this PR isn't necessary any more? But of course I still need to go through and update all of the physical type specifications in the docstrings... Let me know if you disagree and we can re-open this!

**Comment by nstarman** [comment]
Agreed. #243 does exactly what of sphinx-astropy intends. The only thing is that sphinx-astropy needs to be >= v1.4.

**Pull request closed without merging**